cmd/dbaas_show_kafka.go

```diff
@@ -194,17 +190,34 @@ func (c *dbaasServiceShowCmd) showDatabaseServiceKafka(ctx context.Context) (out
 		return &out, nil
 
 	case c.ShowSettings != "":
-		var serviceSettings *map[string]interface{}
 
 		switch c.ShowSettings {
 		case "kafka":
-			serviceSettings = databaseService.KafkaSettings
+			out, err := json.MarshalIndent(databaseService.KafkaSettings, "", "  ")
+			if err != nil {
+				return nil, fmt.Errorf("unable to marshal JSON: %w", err)
+			}
+			fmt.Println(string(out))
```

Comment:
You can de-duplicate this line but moving it after `switch` (and initializing `out` var before).